public/js/save.js

```diff
@@ -124,26 +170,36 @@ $(document).ready(function() {
         counter++;
     }
     //Push All Data
-    function pushData() { //TODO: Check for connectivity before running.
+    function pushData() {
         var error = false;
-        for (var i = (counter-1); i >= 1; i--) {
-            var data = JSON.parse(get('form', i));
-            var request = $.ajax({
-                type: 'POST',
-                url: '/pushData',
-                data: data
-            });
-            request.done(function(response) { //If pushing is successful.
-                console.log("Data successfully pushed.");
-            });
-            request.fail(function(jqXHR, textStatus) { //If pushing is unsuccessful.
-                if(!error){
-                    alert("Error while pushing. Please reach out to a qualified individual for assistance. Error Message: " + textStatus);
-                    error = true;
-                }
-            });
+        var dataArray = {};
+        for (var i = 1; i < counter; i++) {
+            dataArray[i] = get('form', i);
         }
-        counter = 1;
-        localStorage.clear();
+        dataArray = JSON.stringify(dataArray);
+        var request = $.ajax({
+            type: 'POST',
+            url: '/pushData',
+            data: {
+                content: dataArray
+            },
+        });
+        request.done(function(response) { //If pushing is successful.
+            if(response == "No Email"){
```

Comment:
What do you mean by making them constants? What's bad about this method?

I'm just not sure exactly what you mean...